streaming_data_types/arrays_wa00.py

```diff
@@ -0,0 +1,123 @@
+from typing import Union, List, NamedTuple
+from datetime import datetime, timezone
+
+import flatbuffers
+import numpy as np
+from streaming_data_types.fbschemas.arrays_wav_00.WaveFormArray import WaveFormArray
+from streaming_data_types.fbschemas.arrays_wav_00.DType import DType
+from streaming_data_types.fbschemas.arrays_wav_00.WaveFormArray import (
+    WaveFormArrayStart,
+    WaveFormArrayAddXData,
+    WaveFormArrayAddYData,
+    WaveFormArrayAddXDataType,
+    WaveFormArrayAddYDataType,
+    WaveFormArrayAddTimestamp,
+    WaveFormArrayEnd,
+)
+from streaming_data_types.utils import check_schema_identifier
+
+FILE_IDENTIFIER = b"wa00"
+
+
+def serialise_wa00(
+    values_x_array: Union[np.ndarray, List],
+    values_y_array: Union[np.ndarray, List],
+    timestamp: datetime,
```

Comment:
There's a test against `None` below, so this should probably be `Optional[datetime]`. On the other hand, it might be better to remove the `None` test.